ui/src/app/cohort-search/cohort-search/cohort-search.component.spec.tsx

```diff
@@ -53,7 +53,6 @@ describe('CohortSearch', () => {
     serverConfigStore.set({
       config: {
         ...defaultServerConfig,
-        enableDrugWildcardSearch: false,
       },
     });
```

Comment:
Same as above, remove the entire `serverConfigStore` reference from the test.

Reply:
Was removed in updated commit. You are possibly viewing older commit.

Reply:
done